src/ghdcbot/engine/orchestrator.py

```
@@ -0,0 +1,208 @@
+                json_path, _md_path = write_reports(
+                    discord_plans,
+                    github_plans,
+                    self.config,
+                    repo_count=repo_count,
+                    contribution_summaries=contribution_summaries,
+                )
+                logger.info(
+                    "Audit reports written to %s",
+                    str(json_path.parent),
+                )
+            except Exception as exc:  # noqa: BLE001
```

Ruff (0.14.13)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)